src/app/components/Book/styles.js

```diff
@@ -0,0 +1,40 @@
+import {StyleSheet} from 'react-native';
+import {white, black, grey} from '../../../constants/colors'
+
+
+export default StyleSheet.create({
+  container: {
+    flex: 0,
+    height: 90,
+    width: 335,
+    marginHorizontal: 20,
+    flexDirection: 'row',
+    paddingVertical: 15,
+    backgroundColor: white,
+    borderRadius: 5,
+    marginBottom: 10,
+    opacity: 1,
+    shadowColor: black,
+    shadowOffset: {width: 0, height: 1},
+    shadowOpacity: 1,
+    shadowRadius: 1
+  },
+  name: {
+      fontSize: 17,
+      color: black,
+      textAlign: 'left'
+    },
+    author: {
+      fontSize: 15,
+      color: grey,
+      textAlign: 'left'
```

Comment:
i think the color is `#6D6D6D`. Remind me to tell you how to get this colors form invision